05_ANALYSIS_TOOLS/etl_pipeline/config/schemas.yaml

```
@@ -0,0 +1,118 @@
+      units: integer
+      transaction_type: string
+      gross_price: float
+      total_discounts: float
+      net_price: float
+      sales_tax: float
+      total_price: float
+      is_paid: boolean
+      year: integer
+      month: integer
+      quarter: integer
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)